tests/testthat/test-compat-vctrs-parameters.R

```diff
@@ -142,7 +142,16 @@ test_that("vec_cbind() returns a bare tibble", {
   tbl <- tib_upcast(x)
 
   expect_identical(vec_cbind(x), vec_cbind(tbl))
-  expect_identical(vec_cbind(x, x), vec_cbind(tbl, tbl))
-  expect_identical(vec_cbind(x, tbl), vec_cbind(tbl, tbl))
-  expect_identical(vec_cbind(tbl, x), vec_cbind(tbl, tbl))
+  expect_identical(
+    suppressMessages(vec_cbind(x, x)),
+    suppressMessages(vec_cbind(tbl, tbl))
+  )
+  expect_identical(
+    suppressMessages(vec_cbind(x, tbl)),
+    suppressMessages(vec_cbind(tbl, tbl))
+  )
+  expect_identical(
+    suppressMessages(vec_cbind(tbl, x)),
+    suppressMessages(vec_cbind(tbl, tbl))
+  )
```

Comment:
I silenced these since it was quite noisy. but maybe we want to see if anything changes in the messages?

Reply:
What was the message? Probably fine to silence

Reply:
It informs you that it's changing the names of the columns - which I assume is expected? (at least when cbinding x with itself)
```
> vec_cbind(x, x)
New names:
* name -> name...1
* id -> id...2
* source -> source...3
* component -> component...4
* component_id -> component_id...5
* ...
# A tibble: 1 × 12
  name...1 id...2  source...3 component...4 component_id...5 object...6  name...7 id...8  source...9 component...10
  <chr>    <chr>   <chr>      <chr>         <chr>            <list>      <chr>    <chr>   <chr>      <chr>         
1 penalty  penalty list       unknown       unknown          <nparam[+]> penalty  penalty list       unknown       
# … with 2 more variables: component_id...11 <chr>, object...12 <list>
```